randovania/games/samus_returns/json_data/Area 2 - Interior.txt

```diff
@@ -27,7 +27,7 @@ Extra - asset_id: collision_camera009
   * Extra - start_point_actor_name: ST_SaveStation
   > Elevator to Area 2 - Exterior
       Any of the following:
-          Super Jump (Intermediate) or Can Spiderspark or Climb Rooms Vertically (High Jump)
+          High Jump Boots or Space Jump or Super Jump (Intermediate) or Can Spiderspark or Simple IBJ
```

Comment:
I assume the orange line on the top is goo that preevnts you from doing this?
![grafik](https://github.com/randovania/randovania/assets/38186597/4a1a7941-8d08-4b6b-b4d3-469f1d71b9be)

Reply:
Yes